src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");

```

Comment:
The message 'Excavator started work' could be more informative. According to the checklist, the message should indicate what type of `Machine` is working right now. Consider providing more details about the work the Excavator is doing.